includes/class-sensei-utils.php

```diff
@@ -411,6 +411,7 @@ public static function sensei_text_editor( $content = '', $editor_id = 'senseite
 			'editor_class'  => 'sensei_text_editor',
 			'teeny'         => false,
 			'dfw'           => false,
+			'editor_css'    => '<style> .mce-top-part button { background-color: rgba(0,0,0,0.0) !important; } </style>',
```

Comment:
You don't need the `!important` here as it works just fine without it, and leaving it in will make it difficult for others to override.

Also a minor nit, but `0` can be used in place of `0.0` (i.e. `rgba(0, 0, 0, 0)`).

Reply:
Thanks for your comments! 
I just implemented the modifications you suggested...